lib/rack/test.rb

```diff
@@ -240,6 +240,7 @@ def env_for(path, env)
       def process_request(uri, env)
         uri = URI.parse(uri)
         uri.host ||= @default_host
+        uri.scheme ||= "https" if env["HTTPS"] == "on"
```

Comment:
I wonder if we should go with `"true"` and `"false"` here instead of on and off? What do you think @neilang? /cc @junaruga